src/app/views/home/sections/Landing.js

```diff
@@ -13,9 +15,15 @@ function Landing() {
 			<div className="intro-wrapper">
 				<div className="title-info">
 					<Fade duration={1000} left>
+						<div className="jeep-wrapper">
+							<img src={jeepBody} />
+							<img src={jeepWheels} />
```

Comment:
Let's make sure images have meaningful alt text

Reply:
I have added the alt text to the images.